src/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+    crossorigin="anonymous"
+    referrerpolicy="no-referrer">
+    <title>Getting Started</title>
+    <script src="https://unpkg.com/lodash@4.17.20"></script>
+  </head>
+  <body>
+    <main class="main">
+        <div class="container">
+            <div class="list-head">
+            <h1 class="container-h">Today's to-do</h1>
+            <i class="fa-solid fa-arrows-rotate"></i>
+            </div>
+            <hr>
+            <form id="add-task-form">
+                <input type="text" placeholder="Add to your list..." class="add-task">
+            </form>
```

Comment:
- [OPTIONAL] Please consider adding the submit button as shown in the design 

![image](https://user-images.githubusercontent.com/68030297/228381557-9bdbec3d-ad3d-4f04-a654-643feb159949.png)